evm/src/main/java/org/hyperledger/besu/evm/gascalculator/BerlinGasCalculator.java

```diff
@@ -161,8 +192,8 @@ public long callOperationGasCost(
             outputDataLength,
             transferValue,
             recipient,
-            to);
-    final boolean accountIsWarm = frame.warmUpAddress(to) || isPrecompile(to);
+            to,
+            true);
```

Comment:
shouldn't this use the accountIsWarm param that's passed in?

Reply:
No, we want the "warmed price" as we will charge for warming ourselves.  Added a comment to the code.